README.md

```diff
@@ -14,6 +14,25 @@ The connector supports:
 - batching;
 - delivery retries.
 
+An example of configuration:  
```

Comment:
The change was not fully applied. Linting is failing because of trailing white spaces:

```suggestion
An example of configuration:
```